content/issues/2024-03-01-zotero.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'New Container: Zotero'
-date: '2024-02-01 06:00:00Z'
+date: '2024-03-01 06:00:00Z'
```

Comment:
msg in discord  https://discord.com/channels/354974912613449730/512708069709316137/1203496622621724772